.github/workflows/build_unstable_image.yml

```diff
@@ -0,0 +1,41 @@
+name: Build and push image
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  IMG_REGISTRY_HOST: quay.io
+  IMG_REGISTRY_ORG: kuadrant
+  IMG_TAGS: unstable
+
+jobs:
+  build:
+    name: Build and push image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+      - name: Build Image
+        id: build-image
+        uses: redhat-actions/buildah-build@v2
+        with:
+          image: testsuite
```

Comment:
I see REGISTRY_HOST and REGISTRY_ORG even TAG are env vars, I'd personally turn also image name to envvar.